core/ErrorHandler.php

```diff
@@ -39,10 +39,11 @@ class ErrorHandler
      * If a fatal error occurs, theMethodIWantToAppearInFatalErrorStackTraces will appear in the stack trace,
      * if PIWIK_PRINT_ERROR_BACKTRACE is true.
      */
-    public static function pushFatalErrorBreadcrumb($className = null)
+    public static function pushFatalErrorBreadcrumb($className = null, $importantArgs = null)
     {
         $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, $limit = 2);
         $backtrace[1]['class'] = $className; // knowing the derived class name is far more useful
+        $backtrace[1]['args'] = empty($importantArgs) ? [] : array_map('strval', $importantArgs);
```

Comment:
would you need to json_encode the args or so in case it is an array of an array or so?